### PR TITLE
Engdesk 8184 dotnet ivr

### PR DESCRIPTION
**Pull request by d-telnyx**

### CODE REVIEW

**Comment by drewkitch** [comment]
missing .env.sample

File: asp.net-messaging/README.md [hunk 1]
```diff
@@ -18,8 +18,7 @@ You will need to set up:
 
 * [Telnyx Account](https://telnyx.com/sign-up?utm_source=referral&utm_medium=github_referral&utm_campaign=cross-site-link)
 * [Telnyx Phone Number](https://portal.telnyx.com/#/app/numbers/my-numbers?utm_source=referral&utm_medium=github_referral&utm_campaign=cross-site-link) enabled with:
-  * [Telnyx Call Control Application](https://portal.telnyx.com/#/app/call-control/applications?utm_source=referral&utm_medium=github_referral&utm_campaign=cross-site-link)
-  * [Telnyx Outbound Voice Profile](https://portal.telnyx.com/#/app/outbound-profiles?utm_source=referral&utm_medium=github_referral&utm_campaign=cross-site-link)
+  * [Telnyx Messaging Profile](https://portal.telnyx.com/#/app/messaging)
 * Ability to receive webhooks (with something like [ngrok](https://developers.telnyx.com/docs/v2/development/ngrok?utm_source=referral&utm_medium=github_referral&utm_campaign=cross-site-link))
 * [DotNet Core](https://developers.telnyx.com/docs/v2/development/dev-env-setup?lang=java&utm_source=referral&utm_medium=github_referral&utm_campaign=cross-site-link) installed
```
Comment (drewkit) [inline asp.net-messaging/README.md]: link is to java

File: asp_net-IVR/README.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+### Run
+
+Open your IDE and run the application and call the phone number associated with your call control application.
+
+You'll be prompted to enter a 10 digit phone number then the app will transfer to the number entered.
+
+## API Docs
+
+
+## Next Steps
```
Comment (drewkit) [inline asp_net-IVR/README.md]: remove?

File: asp_net-IVR/README.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+                              0       0       0.00    0.00    0.00    0.00
+```
+
+At this point you can point your application to generated ngrok URL + path  (Example: `http://{your-url}.ngrok.io/call-control/inbound`).
+
+### Run
+
+Open your IDE and run the application and call the phone number associated with your call control application.
+
+You'll be prompted to enter a 10 digit phone number then the app will transfer to the number entered.
```
Comment (drewkit) [inline asp_net-IVR/README.md]: Should I be attempting to run this in Visual Studio or IntelliJ IDEA?

Gave it a go with just `dotnet`:

```
drew@Drews-MacBook-Pro asp_net-IVR % dotnet run
Building...
info: Microsoft.Hosting.Lifetime[0]
      Now listening on: http://[::]:80
info: Microsoft.Hosting.Lifetime[0]
      Now listening on: https://localhost:8001
info: Microsoft.Hosting.Lifetime[0]
      Application started. Press Ctrl+C to shut down.
info: Microsoft.Hosting.Lifetime[0]
      Hosting environment: Development
info: Microsoft.Hosting.Lifetime[0]
      Content root path: /Users/drew/repos/team-telnyx/demo-dotnet-telnyx/asp_net-IVR
```

Unclear how I should proceed to trigger prompt for phone number.

File: asp_net-IVR/README.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+## Pre-Reqs
+
+You will need to set up:
+
+* [Telnyx Account](https://telnyx.com/sign-up?utm_source=referral&utm_medium=github_referral&utm_campaign=cross-site-link)
+* [Telnyx Phone Number](https://portal.telnyx.com/#/app/numbers/my-numbers?utm_source=referral&utm_medium=github_referral&utm_campaign=cross-site-link) enabled with:
+  * [Telnyx Call Control Application](https://portal.telnyx.com/#/app/call-control/applications?utm_source=referral&utm_medium=github_referral&utm_campaign=cross-site-link)
+  * [Telnyx Outbound Voice Profile](https://portal.telnyx.com/#/app/outbound-profiles?utm_source=referral&utm_medium=github_referral&utm_campaign=cross-site-link)
+* Ability to receive webhooks (with something like [ngrok](https://developers.telnyx.com/docs/v2/development/ngrok?utm_source=referral&utm_medium=github_referral&utm_campaign=cross-site-link))
+* [DotNet Core](https://developers.telnyx.com/docs/v2/development/dev-env-setup?lang=java&utm_source=referral&utm_medium=github_referral&utm_campaign=cross-site-link) installed
```
Comment (drewkit) [inline asp_net-IVR/README.md]: link is to java

File: asp_net-IVR/README.md (hunk 3 above)
Comment (d-telnyx) [reply asp_net-IVR/README.md]: added clarity, in this case it's triggered by an inbound call.

**Comment by d-telnyx** [comment]
@drewkitch check now

**Comment by drewkit** [comment]
@d-telnyx my ngrok'd webhooks are being treated to a 502 response.. Any idea what I'm doing wrong here?

image

- `ngrok` is listening on port 8000
- call control application is set to fire off webhooks to `http://df66e2aa156c.ngrok.io/call-control/inbound`

**Pull request merged**